src/fem/fe_values.cc

```diff
@@ -552,6 +554,20 @@ void FEValues<dim,spacedim>::reinit(ElementAccessor<3> & cell)
 
 
 
+MixedPtr<FEValues> mixed_fe_values(
+        Mixed<MappingP1> &mapping,
+        Mixed<Quadrature> &quadrature,
+        MixedPtr<FiniteElement> fe,
+        UpdateFlags flags)
+{
+	std::shared_ptr< FiniteElement<0> > fe0 = std::make_shared< FE_P<0> >(0);
+    return MixedPtr<FEValues>(
+      std::make_shared<FEValues<0>>(mapping.get<0>(), quadrature.get<0>(), *fe0, flags),
```

Comment:
Try to resolve FeRT0<0> issue and make this code consistent for dim 0.
If can not be simply resolved make a note in the issue #1003.